README.md

```diff
@@ -1,5 +1,92 @@
 # Serverless Workflow Specification - Typescript SDK
 
-Provides (TODO: add specifics) for the [Serverless Workflow Specification](https://github.com/serverlessworkflow/specification)
+Provides the Java API/SPI for the [Serverless Workflow Specification](https://github.com/serverlessworkflow/specification)
+
+
+With the SDK you can:
+* Parse workflow JSON and YAML definitions
+* (_WIP_) Programmatically build workflow definitions
+
+
+## Getting Started
+
+
+### Building locally
+
+To build the project and run tests locally:
+
+```
+git clone https://github.com/serverlessworkflow/sdk-typescript.git
```

Comment:
can you add second step of 
"cd sdk-typescript" before the npm install...i know its obvious but just in case :)